pkg/reconciler/pipelinerun/resources/pipelineref_test.go

```diff
@@ -576,27 +582,19 @@ func TestGetPipelineFunc_VerifySuccess(t *testing.T) {
 		expected:                  unsignedPipeline,
 		expectedRefSource:         noMatchPolicyRefSource,
 	}, {
-		name:                      "ignore unsigned pipeline without matching policies",
-		requester:                 requesterUnmatched,
-		verificationNoMatchPolicy: config.IgnoreNoMatchPolicy,
+		name:                      "unsigned pipeline fails warn mode policies doesn't return error",
+		requester:                 requesterUnsignedMatched,
+		verificationNoMatchPolicy: config.FailNoMatchPolicy,
 		pipelinerun:               pr,
 		policies:                  vps,
 		expected:                  unsignedPipeline,
-		expectedRefSource:         noMatchPolicyRefSource,
+		expectedRefSource:         warnPolicyRefSource,
 	}, {
-		name:                      "warn no policies",
-		requester:                 requesterUnmatched,
-		verificationNoMatchPolicy: config.WarnNoMatchPolicy,
-		pipelinerun:               pr,
-		policies:                  []*v1alpha1.VerificationPolicy{},
-		expected:                  unsignedPipeline,
-		expectedRefSource:         noMatchPolicyRefSource,
-	}, {
-		name:                      "ignore no policies",
```

Comment:
This is removed as suggested https://github.com/tektoncd/pipeline/pull/6691#discussion_r1204467614, same for taskref_test.go

Reply:
Thanks for the clarifications!